### PR TITLE
Add size in days

### PR DESCRIPTION
**Pull request by lucacervello**

### CODE REVIEW

File: src/Onebip/DateTime/UTCDateTimeRange.php [hunk 1]
```diff
@@ -168,6 +168,11 @@ public function direction()
         }
     }
 
+    public function sizeInDays()
```
Comment (pasquinis) [inline src/Onebip/DateTime/UTCDateTimeRange.php]: what do you think about a nane like `distanceInDays()`?

**Review by pasquinis — CHANGES_REQUESTED** [review]

File: src/Onebip/DateTime/UTCDateTimeRange.php (hunk 1 above)
Comment (lucacervello) [reply src/Onebip/DateTime/UTCDateTimeRange.php]: We are talking about `range`, `size` sound better to me. Maybe `interval` is better with dates.

Comment (pasquinis) [reply src/Onebip/DateTime/UTCDateTimeRange.php]: or simply `differenceBetweenDatesInDays`

Comment (lucacervello) [reply src/Onebip/DateTime/UTCDateTimeRange.php]: It's a public function i would like to keep the name nice and short. I think `intervalInDays()` it the best candidate right now.

Comment (pasquinis) [reply src/Onebip/DateTime/UTCDateTimeRange.php]: Good @lucacervello  you are right

**Review by pasquinis — APPROVED** [review]

**Pull request merged**